src/languages/de.json

```diff
@@ -0,0 +1,101 @@
+{
+    "chat": {
+        "actions": {
+            "banUser": "Ban Benutzer",
+            "makeMod": "Promote zu Moderator",
+            "makeAdmin": "Promote zu Admin",
+            "makeUser": "Promote zu Benutzer",
+            "close": "Schlie�en",
+            "invite": "Zum Chat einladen ",
+            "createChat": "Chat erstellen",
+            "joinChat": "Chat beitreten",
+            "createAddChat": "Klicke, um neuen Chat zu erstellen",
+            "copyIdPasswort": "Kopiere Passwort&Id",
+            "deleteRoom": "Raum l�schen",
+            "leaveRoom": "Raum verlassen"
+        },
+        "texts": {
+            "settings": "Mehr Einstellungen bei ",
+            "account": "Account",
+            "title": "Chat",
+            "creditEmoticon": "Emoticons bereitgestellt durch ",
+            "optional": "Dieser Teil des Skripts ist optional, einloggen ist nicht n�tig, um beispielsweise Raids beizutreten",
+            "invalid": "Sie sollten nicht Ihre Login-Daten, wie Name oder Passwort der Website oder des Spiels, benutzen - der angezeigte Name kann nach der Registrierung angepasst werden.",
+            "offline": "Sie sind nicht eingeloggt, bitte registrieren oder loggen Sie sich ein"
+        },
+        "error": {
+            "modify": "Konnte Benutzer nicht modifizieren",
+            "create": "Erstellung fehlgeschlagen",
+            "join": "Beitritt fehlgeschlagen",
+            "login": "Login auf unerwartete Weise fehlgeschlagen",
+            "unknown": "Der eingegebene Benutzername ist dotd.idrinth.de nicht bekannt, m�chten Sie sich dort registrieren?"
+        },
+        "notification": {
```

Comment:
It is unclear to me how to translate this term. Something like "Benachrichtigungen für Chat-Nachrichten" ?